src/styles/blocks/main.css

```diff
@@ -0,0 +1,50 @@
+main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  contain: content;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  margin: 0 22px;
+}
+
+.stars__star {
+  background-image: url(../../images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-left: 4px;
+  margin-bottom: 4px;
+  border-radius: 0.5px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(n + 2) {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--2 .stars__star:nth-child(n + 3) {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--3 .stars__star:nth-child(n + 4) {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(n + 5) {
+  background-image: url(../../images/star.svg);
```

Comment:
you could comprise all of these selectors in a single line - try to think of a way to do that